### DISCUSSION TITLE
Does CustomTkinter Work with Window 7 and Python 3.7

### DISCUSSION THREAD

**Discussion by H1Mohamed** [TomSchimansky/CustomTkinter#1074] [General]
I'm a python & Tkinter dev using window 7 and the latest version of python I was able to install (Python 3.7.0),
So does it work

**Comment by TomSchimansky** [comment]
I don't know, just try it.

**Comment by RutgerLit** [comment]
No it won't. Ctk uses windll.shcore which is not available in Windows 7. You have to adjust the ctk sourcecode. For example, for dpi awareness use

```
try: # >= win 8.1
    windll.shcore.SetProcessDpiAwareness(2)
except: # win 8.0 or less
    windll.user32.SetProcessDPIAware()
```
